src/main/java/com/tagtraum/perf/gcviewer/ctrl/action/Export.java

```diff
@@ -96,31 +96,31 @@ public void exportFile(final GCModel model, File file, final String extension, f
         }
     }
 
-    private static class ExtensionFileFilter extends FileFilter {
-        private String extension;
-        private String description;
-        private DataWriterType dataWriterType;
+    private static class ExportExtensionFileFilter extends ExtensionFileFilter {
+
+        public static final ExportExtensionFileFilter[] EXT_FILE_FILTERS = {
+            new ExportExtensionFileFilter(".csv", LocalisationHelper.getString("fileexport_dialog_csv"), DataWriterType.CSV),
```

Comment:
To confirm with the "accept()" implementation of ExtensionFileFilter you need to remove the "." from the file extensions here. The implementation of the deleted and the new "accept()" differs in exactly this point. Otherwise no files are shown any more in the export dialog.